src/platform/i386/timer.c

```diff
@@ -131,6 +145,9 @@ timer_calibration(void)
 		timer_cycles_per_hpetcyc = (TIMER_ERROR_BOUND_FACTOR * cycles_per_tick) / hpetcyc_per_tick;
 		printk("Timer calibrated:\n\tCPU cycles per HPET tick: %ld\n\tHPET ticks in %d us: %ld\n",
 		       timer_cycles_per_hpetcyc/TIMER_ERROR_BOUND_FACTOR, TIMER_DEFAULT_US_INTERARRIVAL, hpetcyc_per_tick);
+
+		timer_disable(TIMER_PERIODIC);
+		timer_disable(TIMER_PERIODIC);
```

Comment:
we'll likely want to figure out which part of the programming requires the double programming, and only do that.  I talked to one of the L4 guys, and he said the hpet is horrible.